bitcoin/src/address.rs

```diff
@@ -469,6 +469,28 @@ impl Payload {
         }
     }
 
+    /// Returns true if the address creates a particular script
```

Comment:
A comment about not allocating would be helpful.

Reply:
thanks, added